### PR TITLE
Spring cleaning for tasks 🧹🧼 

### PR DESCRIPTION
**Pull request by merveenoyan**
Updating all pages (mostly text generation, image generation and popular ones) for existing paradigm, adding state of the art models and Spaces for each page and more.

### CODE REVIEW

**Comment by merveenoyan** [comment]
@osanseviero this PR is ready for review (also got it up on codespaces, seems to be fine)

File: tasks/src/image-to-text/data.ts [hunk 1]
```diff
@@ -39,6 +39,30 @@ const taskData: TaskDataCustom = {
...
+			description: "A visual question answering model for book covers.",
+			id:          "google/pix2struct-ocrvqa-large",
+		},
+		{
+			description: "A visual question answering model for charts.",
+			id:          "google/pix2struct-chartqa-base",
+		},
+		{
+			description: "A captioning model for images that contain text.",
+			id:          "google/pix2struct-textcaps-base",
```
Comment (osanseviero) [inline tasks/src/image-to-text/data.ts:64]: Aren't these too many models?

File: tasks/src/text-generation/about.md [hunk 2]
```diff
@@ -1,12 +1,18 @@
+This task covers guides on both [text-generation](https://huggingface.co/models?pipeline_tag=text-generation&sort=downloads) and [text-to-text generation](https://huggingface.co/models?pipeline_tag=text2text-generation&sort=downloads) models. Popular large language models that are used for chats or following instructions are also covered in this task. You can find the list of all open-source large language models [here](https://huggingface.co/spaces/HuggingFaceH4/open_llm_leaderboard), ranked by their performance scores.
```
Comment (osanseviero) [inline tasks/src/text-generation/about.md:1]:
```suggestion
This task covers guides on both [text-generation](https://huggingface.co/models?pipeline_tag=text-generation&sort=downloads) and [text-to-text generation](https://huggingface.co/models?pipeline_tag=text2text-generation&sort=downloads) models. Popular large language models that are used for chats or following instructions are also covered in this task. You can find the list of selected open-source large language models [here](https://huggingface.co/spaces/HuggingFaceH4/open_llm_leaderboard), ranked by their performance scores.
```

File: tasks/src/text-generation/about.md [hunk 3]
```diff
@@ -1,12 +1,18 @@
...
 ## Use Cases
 
+### Instruction Models
+
+A model trained for text generation can be later adapted to follow instructions. One of the most used open-source models for instruction is OpenAssistant, which you can try [at Hugging Chat](https://huggingface.co/chat).
+
 ### Code Generation
 
-A Text Generation model, also known as causal language model, can be trained on code from scratch to help the programmers in their repetitive coding tasks.
+A Text Generation model, also known as causal language model, can be trained on code from scratch to help the programmers in their repetitive coding tasks. One of the most used open-source models for code generation is StarCoder, can generate code in 80+ languages. You can try it [here](https://huggingface.co/spaces/bigcode/bigcode-playground).
```
Comment (osanseviero) [inline tasks/src/text-generation/about.md:11]:
```suggestion
A Text Generation model, also known as a causal language model, can be trained on code from scratch to help the programmers in their repetitive coding tasks. One of the most popular open-source models for code generation is StarCoder, which can generate code in 80+ languages. You can try it [here](https://huggingface.co/spaces/bigcode/bigcode-playground).
```

File: tasks/src/text-generation/data.ts [hunk 4]
```diff
@@ -46,8 +46,48 @@ const taskData: TaskDataCustom = {
...
+			description: "A large language model trained to generate text in English.",
+			id:          "EleutherAI/pythia-12b",
+		},
+		{
+			description: "A large language model trained to follow instructions in English.",
+			id:          "togethercomputer/RedPajama-INCITE-Instruct-7B-v0.1",
+		},
+		{
+			description: "A large text-to-text model trained to follow instructions.",
+			id:          "google/flan-ul2",
```
Comment (osanseviero) [inline tasks/src/text-generation/data.ts:90]: Same as above, these might be too many models and might not look super good in mobile

**Review by osanseviero — APPROVED** [review]
Lots of cool things! Thanks a lot :rocket:

File: tasks/src/feature-extraction/data.ts [hunk 5]
```diff
@@ -43,9 +43,13 @@ const taskData: TaskDataCustom = {
 			id:          "facebook/bart-base",
 		},
 		{
-			description: "Pretrained weights for CodeBERT: A Pre-Trained Model for Programming and Natural Languages.",
+			description: "A strong feature extraction model for coding tasks.",
 			id:          "microsoft/codebert-base",
 		},
+		{
+			description: "A feature extraction model for images.",
```
[the remark this answers is not in the document]
Comment (merveenoyan) [reply tasks/src/feature-extraction/data.ts:50]: I feel like it's more about _embeddings_, no? You can still take image embeddings and put a classifier on top of it. WDYT?

File: tasks/src/image-to-text/data.ts (hunk 1 above)
Comment (merveenoyan) [reply tasks/src/image-to-text/data.ts:64]: I removed VQA ones.

File: tasks/src/image-to-image/about.md [hunk 6]
```diff
@@ -21,11 +21,28 @@ Super resolution models increase the resolution of an image, allowing for higher
...
+prompt = "A fantasy landscape, trending on artstation"
+
+images = pipe(prompt=prompt, image=init_image, strength=0.75, guidance_scale=7.5).images
+images[0].save("fantasy_landscape.png")
+```
+
+## ControlNet
 
-You can contribute useful resources about this task [here](https://github.com/huggingface/hub-docs/blob/main/tasks/src/image-to-image/about.md).
+Controlling outputs of diffusion models only with a text prompt is a challenging problem. ControlNet is a neural network type that provides an image based control to diffusion models. These controls can be edge detection outputs, canny filter, or landmarks.
```
[the remark this answers is not in the document]
Comment (merveenoyan) [reply tasks/src/image-to-image/about.md:43]: Just made it less technical

File: tasks/src/image-segmentation/data.ts [hunk 7]
```diff
@@ -64,8 +80,16 @@ const taskData: TaskDataCustom = {
...
+			description: "An application that segments videos.",
+			id:          "ArtGAN/Segment-Anything-Video",
+		},
+		{
+			description: "An panoptic segmentation application built for outdoor environments.",
+			id:          "segments/panoptic-segment-anything",
+		},
+		{
+			description: "An panoptic segmentation application built for outdoor environments.",
+			id:          "segments/panoptic-segment-anything",
```
[the remark this answers is not in the document]
Comment (merveenoyan) [reply tasks/src/image-segmentation/data.ts:92]: removed

File: tasks/src/feature-extraction/data.ts (hunk 5 above)
[the remark this answers is not in the document]
Comment (merveenoyan) [reply tasks/src/feature-extraction/data.ts:50]: Removed for now

**Comment by younver** [comment]
🎉

**Pull request merged**